cmd/thanos/main.go

```diff
@@ -48,7 +53,6 @@ func main() {
 	registerCompact(app)
 	registerTools(app)
 	registerReceive(app)
-	registerReceiveRoute(app)
```

Comment:
We still need this

Reply:
My bad, I kind of reverted all the changes of `main.go`